### PR TITLE
1.20 port

### PR DESCRIPTION
**Pull request by gohkenytp**
Ported this mod to Minecraft 1.20

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -18,17 +18,17 @@ apply plugin: 'org.spongepowered.mixin'
 apply plugin: 'org.parchmentmc.librarian.forgegradle'
 apply plugin: 'eclipse'
 
-version = '1.25'
+version = '1.26'
```
Comment (bl4ckscor3) [inline build.gradle:21]: No version increase for a Minecraft update.

File: build.gradle [hunk 2]
```diff
@@ -18,17 +18,17 @@ apply plugin: 'org.spongepowered.mixin'
...
 group = 'bl4ckscor3.mod.ceilingtorch'
-archivesBaseName = 'ceilingtorch-1.19.3'
+archivesBaseName = 'ceilingtorch-1.20'
 
 java.toolchain.languageVersion = JavaLanguageVersion.of(17)
 println('Java: ' + System.getProperty('java.version') + ' JVM: ' + System.getProperty('java.vm.version') + '(' + System.getProperty('java.vendor') + ') Arch: ' + System.getProperty('os.arch'))
 
 minecraft {
-    mappings channel: 'parchment', version: '2022.12.18-1.19.3'
+    mappings channel: 'official', version: '1.20'
```
Comment (bl4ckscor3) [inline build.gradle:30]: Please use parchment. I am aware it is not updated to 1.20 yet, but I prefer incomplete parameter mappings over none.

File: src/main/resources/META-INF/mods.toml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 modLoader="javafml"
-loaderVersion="[44,)"
+loaderVersion="[46,)"
 license="MIT https://github.com/bl4ckscor3/CeilingTorch/blob/1.19/LICENSE"
```
Comment (bl4ckscor3) [inline src/main/resources/META-INF/mods.toml]: Update the branch to `1.20`, please.

**Review by bl4ckscor3 — CHANGES_REQUESTED** [review]
Thank you! This still needs changes which I have outlined below and in review comments.

- Upgrade the Gradle wrapper and relevant scripts to 8.1.1. Best to just copy the files from the Forge MDK.
- Please apply this change https://github.com/Geforce132/SecurityCraft/commit/f4d2a30c73c2a64c139d329c3a6f95dfb0d1f990#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7L26-R29
- Same with this one https://github.com/Geforce132/SecurityCraft/commit/f4d2a30c73c2a64c139d329c3a6f95dfb0d1f990#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R53
- Please add this https://github.com/Geforce132/SecurityCraft/commit/f4d2a30c73c2a64c139d329c3a6f95dfb0d1f990#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R139-R141
- The pack.mcmeta needs to be updated. It should be as seen below.
```json
{
  "pack": {
    "description": {
      "text": "Mod resources & data"
    },
    "pack_format": 15,
    "forge:server_data_pack_format": 15
  }
}
```

**Review by gohkenytp — COMMENTED** [review]
I already ported this to 1.20

**Review by gohkenytp — COMMENTED** [review]
Look, not to be rude, but I ported your Ceiling Torch mod to 1.20 fair and square. Glad you like the idea.

**Comment by bl4ckscor3** [comment]
I don't understand your most recent comment. I appreciate you porting the mod, and in the usual way how pull requests go, I asked for changes to be made before the PR gets merged. There's no ill intent from my side.

I will be reviewing this again tomorrow as it's getting late here.

**Comment by gohkenytp** [comment]
OK. I sure hope this goes successful!

File: build.gradle (hunk 1 above)
Comment (bl4ckscor3) [reply build.gradle:21]: This still needs to be addressed.

File: build.gradle (hunk 2 above)
Comment (bl4ckscor3) [reply build.gradle:30]: This still needs to be addressed.

File: src/main/resources/META-INF/mods.toml (hunk 3 above)
Comment (bl4ckscor3) [reply src/main/resources/META-INF/mods.toml]: This still needs to be addressed.

**Comment by bl4ckscor3** [comment]
From my previous review comment, 4 points have not been addressed, as well as the comments I left on the code itself.

- Upgrade the Gradle wrapper and relevant scripts to 8.1.1. Best to just copy the files from the Forge MDK.
- Please apply this change https://github.com/Geforce132/SecurityCraft/commit/f4d2a30c73c2a64c139d329c3a6f95dfb0d1f990#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7L26-R29
- Same with this one https://github.com/Geforce132/SecurityCraft/commit/f4d2a30c73c2a64c139d329c3a6f95dfb0d1f990#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R53
- Please add this https://github.com/Geforce132/SecurityCraft/commit/f4d2a30c73c2a64c139d329c3a6f95dfb0d1f990#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R139-R141

**Comment by gohkenytp** [comment]
Ok I did it already.

**Comment by bl4ckscor3** [comment]
Not everything is addressed yet.

**Comment by gohkenytp** [comment]
Look, I tried to port it, but you didn't seem to understand.

**Comment by bl4ckscor3** [comment]
I do understand. And I can just reiterate that I am happy you did the port. But it is lacking some necessary changes for it to be considered complete. I requested those changes to be made, that's how pull requests work. If you don't want to finish the port it's fine, I'll base mine off of yours once I get around to doing it. But unless you can address my previous comments, this is not getting merged.

**Comment by gohkenytp** [comment]
I don't want to finish this port sadly.

**Comment by gohkenytp** [comment]
Also, why are you halting progress on your port to 1.20 anyway?

**Comment by bl4ckscor3** [comment]
Please see my comment on the other repository. There's no reason to crosspost.

**Comment by gohkenytp** [comment]
Not to be rude but, I just can't do it now. OK? Accept it. Maybe you can port it without me if you get a chance.

**Pull request closed without merging**